tests/test_pfs.py

```diff
@@ -369,66 +369,154 @@ def test_wait_commit():
     """
     Ensure wait_commit works
     """
-
     client, repo_name = sandbox("wait_commit")
+    repo2_name = util.create_test_repo(client, "wait_commit2")
+
+    # Create provenance between repos (which counts as a commit)
+    client.create_branch(
+        repo2_name,
+        "master",
+        provenance=[
+            pfs_proto.Branch(
+                repo=pfs_proto.Repo(name=repo_name, type="user"), name="master"
+            )
+        ],
+    )
+    # Head commit is still open in repo2
+    client.finish_commit((repo2_name, "master"))
 
     with client.commit(repo_name, "master") as c:
         client.put_file_bytes(c, "input.json", b"hello world")
+    client.finish_commit((repo2_name, "master"))
 
     # Just block until all of the commits are yielded
-    assert len(list(client.wait_commit(c.id))) == 1
-    assert len(list(client.wait_commit(c))) == 1
+    assert len(client.wait_commit(c.id)) == 2
+    c_downstream = list(
+        client.inspect_commit((repo2_name, "master"), pfs_proto.CommitState.FINISHED)
+    )[0]
+    assert c_downstream.finished
```

Comment:
I'm not sure I understand what this assertion tests, but would it make sense to just capture the output of `wait_commit`? Like:
```
commits = client.wait_commit(c.id)
assert len(commits) == 2
assert commits[0].finished
```

Alternative suggestion: since `c_downstream` doesn't depend on `wait_commit`, would it make sense to break it out into a different test? (IMO smaller tests are generally easier to maintain, and easier to fix when they break)

Reply:
Yep, you're right, `wait_commit` returns us the same information that the `inspect_commit` returns us. Updating...